src/app/interfaces.ts

```diff
@@ -162,6 +162,52 @@ export interface IUserTagCategory {
     tags: IUserTag[];
 }
 
+export interface IComplaintTag {
+    id: string;
+    tag_uri: string;
+}
+
+export interface IComplaintComment {
+    id: string;
+    time: string;
+    text: string;
+    commented_by: IUserProfile;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'commented_by' is not in camel case.](https://app.codacy.com/gh/wncc/instiapp-angular/pullRequest?prid=5455571)